ros/src/waypoint_follower/include/pure_pursuit_core.h

```diff
@@ -70,6 +70,9 @@ class PurePursuit
   bool pose_set_;
   bool velocity_set_;
   int num_of_next_waypoint_;
+  // edufford start
```

Comment:
FYI - `git blame` will show the exact lines that you modified, so we'll always have a history of your changes vs those that were contributed before you. You can also see this by clicking the "blame" button in the github UI.

You don't need to remove this, I just wanted to mention it in case you didn't know that ability existed.

Reply:
Ah, that's a nice feature!  I'll clear the comments.